Describe export to excel restrictions (T852278) (#590)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/remoteOperations/summary.md

> Specifies whether or not [summaries](/concepts/05%20Widgets/DataGrid/65%20Summaries/10%20Total%20Summary '/Documentation/Guide/Widgets/DataGrid/Summaries/Total_Summary/') calculation must be performed on the server.

Reviewer: Can we be more direct (and avoid the word "perform"):
Specifies whether or not summaries are calculated on the server-side.

---

> [note]When [custom summary calculation](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/summary/#calculateCustomSummary) is performed on a server, the [allowExportSelectedData](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/#allowExportSelectedData) option is disabled.

Reviewer 2: Again, "option is disabled." - "option is not supported" looks better for me.

Author: Replied above

Reviewer 3: I think the original note is distorted. calculateCustomSummary is a client-side function. The original note said that this function cannot be used when a remote summary is used. This is now lost.

Author: Do you mean we should just remove this note?

> [note]When summary calculation is remote, the widget does not support [custom summary calculation](/concepts/05%20Widgets/DataGrid/65%20Summaries/07%20Custom%20Aggregate%20Function '/Documentation/Guide/Widgets/DataGrid/Summaries/Custom_Aggregate_Function/') and the [allowExportSelectedData](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/#allowExportSelectedData) option cannot be enabled.

Reviewer 3: ```suggestion
[note]When summary calculation is remote, [custom client-side summaries](/Documentation/Guide/Widgets/DataGrid/Summaries/Custom_Aggregate_Function/#Client-Side_Data_Aggregation) are not supported.
```
